modules/ROOT/pages/procedures.adoc

```diff
@@ -1884,63 +1884,69 @@ The procedures deprecated in Neo4j 5 will be removed in the next major release o
 | xref:procedures.adoc#procedure_cdc_current[`cdc.current()`]
 | label:no[]
 | label:yes[]
-| label:new[Introduced in 5.13] label:beta[] label:deprecated[Deprecated in 5.17]
+| label:beta[] label:deprecated[Deprecated in Cypher 5] label:deprecated[Deprecated in Cypher 25] +
 Replaced by: xref:procedures.adoc#procedure_db_cdc_current[`db.cdc.current()`]
 
 | xref:procedures.adoc#procedure_cdc_earliest[`cdc.earliest()`]
 | label:no[]
 | label:yes[]
-| label:new[Introduced in 5.13] label:beta[] label:deprecated[Deprecated in 5.17]
+| label:beta[] label:deprecated[Deprecated in Cypher 5] label:deprecated[Deprecated in Cypher 25] +
 Replaced by: xref:procedures.adoc#procedure_db_cdc_earliest[`db.cdc.earliest()`]
 
 | xref:procedures.adoc#procedure_cdc_query[`cdc.query()`]
 | label:no[]
 | label:yes[]
-| label:new[Introduced in 5.13] label:beta[] label:admin-only[] label:deprecated[Deprecated in 5.17]
+| label:beta[] label:admin-only[] label:deprecated[Deprecated in Cypher 5] label:deprecated[Deprecated in Cypher 25] +
```

Comment:
```suggestion
| label:beta[] label:admin-only[] label:deprecated[Deprecated since Neo4j 5.17] label:available[Available in Cypher 5 and Cypher 25] +
```